migrations/001_create_tables.sql

```diff
@@ -0,0 +1,34 @@
+--
+-- file migrations/001_create_tables.py
+--
+
+
+CREATE TABLE countries(
+            country_id SERIAL PRIMARY KEY,
+            country VARCHAR(50) UNIQUE
+        );
+
+CREATE TABLE states(
+    state_id SERIAL PRIMARY KEY,
+    state VARCHAR (50) UNIQUE,
+    country_id INTEGER, 
```

Comment:
you should have an index on `country_id` for use in `WHERE` clauses

Reply:
Please, when should I create an index on the columns you pointed out? At the point of creating the tables in the sql script?